`uuid` dependency or dev dependency?

In the latest release `uuid` has been moved from `dependencies` to `devDependencies`. In environments that prune the dev dependencies this then causes an error on running the build:

```
Error: Cannot find module 'uuid'
```

So it looks like `uuid` should be moved back to `dependencies`?

--- a/package.json
+++ b/package.json
@@ -30,6 +30,10 @@
     "package.json",
     "README.md"
   ],
+  "dependencies": {
+    "debug": "^4.3.4",
+    "uuid": "^9.0.1"
+  },
   "peerDependencies": {
     "@remix-run/server-runtime": "^1.0.0 || ^2.0.0",
     "remix-auth": "^3.6.0"
@@ -61,10 +65,6 @@
     "react-dom": "^18.2.0",
     "remix-auth": "^3.6.0",
     "ts-node": "^10.9.1",
-    "typescript": "^5.2.2",
-    "uuid": "^9.0.1"
-  },
-  "dependencies": {
-    "debug": "^4.3.4"
+    "typescript": "^5.2.2"
   }
 }

